src/pages/FormBuilder/FormBuilder.module.css

```
@@ -0,0 +1,871 @@
+  background: var(--blue);
+}
+
+.tabLabel {
+  font-size: 12px;
+  font-weight: 500;
+  color: var(--gray-500);
+  text-align: center;
+  width: 100%;
+  line-height: 1.3;
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)